CHANGES.txt

```diff
@@ -86,7 +86,17 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       still run into problems on some configurations), and automated
       tests are now run on changes in this area so future problems can
       be spotted.
-
+    - Special-case handling for CPPDEFINES in the Append/Prepend routines
```

Comment:
This should probably be multiple bullets.

Reply:
Can do that, but it's really convouted - see the long report in the one issue of the various inconsistencies I had discovered.  So there would be a whole bunch of "now get (b) instead of (a), but only if (c); if (d) the behavior was already (b); if (e), this previously did not work at all".

Reply:
I think that would be valuable.  This whole PR is really about fixing some corner cases (and reorganizing the code so it's more sane), so explaining which corner cases are now fixed (changed) would be useful for anyone who updated and saw their build change, and/or has a build not doing what they expect and sees that the newer version solves that.

Bigger organizations often need a forcing function to move to a newer version.